### DISCUSSION TITLE
Paid Material Themes

### DISCUSSION THREAD

**Discussion by TorreyGarland** [Megabit/Blazorise#3812] [Q&A] [answered]
Is there a theme available using Material?

**Comment by stsrki** [comment, marked as answer]
Not at the moment. But our current theme can easily be changed to use a Material provider. You just need to follow the guide at https://blazorise.com/docs/usage/material and change the bootstrap references and files to use the material.

Here is an example:

![image](https://user-images.githubusercontent.com/900302/168898346-803e4e59-31ba-4f17-8dc6-77d5f1876093.png)

**Comment by TorreyGarland** [comment]
thanks for the feedback.

consider this answered.
